publisher-sdk/src/main/java/com/criteo/publisher/CriteoBannerView.java

```diff
@@ -41,14 +44,39 @@
    * having to init the SDK before.
    */
   @Nullable
-  private final Criteo criteo;
+  private Criteo criteo;
```

Comment:
Same than above. You just need to initialize it to keep it `final`

Reply:
Actually, I don't remember the purpose of this field? When is it initialized? It seems calling getCriteo (https://github.com/criteo/android-publisher-sdk/blob/main/publisher-sdk/src/main/java/com/criteo/publisher/CriteoBannerView.java#L129) will always call `Criteo.getInstance()`, no?

Reply:
It is used for unit tests by giving a Criteo mock directly.